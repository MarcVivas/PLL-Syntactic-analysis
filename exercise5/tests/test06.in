// Errors on lines 2 and 6
S  Ac | fA | B
;
B : b 
;
A  i | a | B 
;
