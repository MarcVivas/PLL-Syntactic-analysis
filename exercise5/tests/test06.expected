Error at line 2: syntax error
Error at line 6: syntax error
B= b


